src/lib/src/datepicker/daterange.component.css

```diff
@@ -0,0 +1,22 @@
+:host {
+  display: flex;
+}
+.picker {
+  position: relative;
+  text-align: center;
+}
+.picker:first-child {
+  margin-right: 20px;
+}
+.picker > label {
```

Comment:
This isn't _exactly_ how the labels are styled in the design ticket https://github.com/PRX/metrics.prx.org/issues/80, but I like it better.

Reply:
this way looks nice